docs/guides/install-c-api.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Install the Qiskit C API
+description: How to install and use the Qiskit C API as standalone library
+---
+
+# Install the Qiskit C API
+
+This guide describes how to install and use the Qiskit C API. To see how to extend your
+Qiskit Python workflow with C, read [Extend Python with the Qiskit C API](/guides/c-extension-for-python).
+
+The following example builds an observable with C:
+```c
+// file: example.c
+#include <stdio.h>
+#include <stdint.h>
+#include <complex.h>
+#include <qiskit.h>
+
+int main(int argc, char* argv[]) {
+    // build a 100-qubit empty observable
+    u_int32_t num_qubits = 100;
+    QkObs *obs = qk_obs_zero(num_qubits);
+
+    // add the term 2 * (X0 Y1 Z2) to the observable
+    complex double coeff = 2;  // on MSVC use: _Dcomplex coeff = {2, 0}
+    QkBitTerm bit_terms[3] = {QkBitTerm_X, QkBitTerm_Y, QkBitTerm_Z};  // bit terms: X Y Z
+    uint32_t indices[3] = {0, 1, 2};  // indices: 0 1 2
+    QkObsTerm term = {coeff, 3, bit_terms, indices, num_qubits};
+    qk_obs_add_term(obs, &term);  // append the term
+
+    // print some properties and the observable itself
+    printf("num_qubits: %i\n", qk_obs_num_qubits(obs));
+    printf("num_terms: %lu\n", qk_obs_num_terms(obs));
+    printf("observable: %s\n", qk_obs_str(obs));
+
+    // free the memory allocated for the observable
+    qk_obs_free(obs);
+
+    return 0;
+}
+```
+
+## UNIX
+
+This section provides build instructions for UNIX systems.
+
+### Requirements
```

Comment:
You might want to point to the platform support page here so people know the platform requirements for building from source.